features/step_definitions/crd.rb

```diff
@@ -29,3 +29,18 @@
     raise "Cannot restore #{crd}: #{name}" unless @result[:success]
   }
 end
+
+Given /^I remove all CRDs with labels:$/ do | table |
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  labels = table.raw.flatten
+  timeout = 60
+  res = BushSlicer::CustomResourceDefinition.wait_for_labeled(*labels, user: user, seconds: timeout)
+  if res[:success]
+    crd_names = res[:parsed]['items'].map { |r| r['metadata']['name'] }
+    crd_names.each do | crd |
+      step %Q/I ensure "#{crd}" custom_resource_definition is deleted/
```

Comment:
shouldn't this be `admin ensures ...`?

Reply:
since this step is tagged with `ensure_admin_tagged`, wouldn't that already implies the `I` in this case is `admin`?

Reply:
@akostadinov I've resolved the conflicts...anything else that needs to be modified before it can merged?

Reply:
> since this step is tagged with ensure_admin_tagged, wouldn't that already implies the I in this case is admin?

no, this is not the case

Reply:
```
list = clazz = resource_class(resource_type)
if BushSlicer::ProjectResource > clazz
  list = clazz.list(*labels, user: user, project: project(namespace))
else
  ensure_destructive_tagged
  list = clazz.list(*labels, user: user)
end
list.each { |r| r.ensure_deleted }
```

Reply:
but making step universal makes this moot